marshmallow/schema.py

```diff
@@ -174,6 +201,37 @@ def _resolve_processors(self):
                 self.__processors__[tag].append(attr_name)
 
 
+class SchemaJit(with_metaclass(ABCMeta)):
```

Comment:
Couldn't this go in your external library?

Ideally, all JIT-related code would go in the library, i.e. `JITSchema`, `JITSchemaOpts`, `SchemaJIT`..

I imagine it would look like:

```python
import os
from marshmallow import Schema, fields
from toastedmarshmallow import Schema as JITSchema, DefaultJIT

BaseSchema = JITSchema if os.environ.get('MARSHMALLOW_USE_JIT') else Schema

class MySchema(BaseSchema):
    class Meta:
        jit = DefaultJIT()
```